app/models/meal.rb

```diff
@@ -0,0 +1,23 @@
+class Meal < ApplicationRecord
+  ### ASSOCIATIONS ###
+
+  has_one_attached :image
+  has_many :ingredients_meals, class_name: 'IngredientMeal', dependent: :destroy
+  has_many :ingredients, through: :ingredients_meals
+  accepts_nested_attributes_for :ingredients_meals, update_only: true, allow_destroy: true
+
+  ### VALIDATIONS ###
+
+  validates :name, presence: true
+  validates :image, content_type: { in: %w[image/jpeg image/jpg image/png] }
+  validates :image, size: { less_than: 5.megabytes }
+  validates :price, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
Getting this error for new meals
<img width="667" alt="image" src="https://github.com/rishabh-vinsol/restaurant-pos/assets/64541606/d85ea29c-719a-475a-9db6-731e7c7007fa">

Reply:
Sir its working fine, please check if it persists in the next PR